### PR TITLE
Lc problem18 remove nth node from end of list

### PR DESCRIPTION
**Pull request by CHAriina**
solving the problem remove-nth-node-from-end-of-list (day 16 point 1)

### CODE REVIEW

File: ScnSoft-Education/LeetCodeTraining.Tests/RemoveNthNodeFromEndOfList.Tests.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
+﻿using LeetCodeTraining.RemoveNthNodeFromEndOfList18;
+
+namespace LeetCodeTraining.Tests.RemoveNthNodeFromEndOfList
+{
+    public class RemoveNthNodeFromEndOfListTests
+    {
+        [Theory]
+        [InlineData(new int[] { 1, 2, 3, 4, 5 }, 2, new int[] { 1, 2, 3, 5 })]
+        [InlineData(new int[] { 1, 2, 3, 4, 5 }, 1, new int[] { 1, 2, 3, 4 })]
```
Comment (dsubotkovsky) [inline ScnSoft-Education/LeetCodeTraining.Tests/RemoveNthNodeFromEndOfList.Tests.cs:10]: add unit test where you want to delete elements wich doesn't exist in your array e.g
[InlineData(new int[] { 1, 2, 3, 4, 5 }, 6, new int[] { 1, 2, 3, 4, 5 })]

File: ScnSoft-Education/LeetCodeTraining/RemoveNthNodeFromEndOfList18/RemoveNthNodeFromEndOfList.cs [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    {
+        public int val;
+        public ListNode next;
+        public ListNode(int val = 0, ListNode next = null)
+        {
+            this.val = val;
+            this.next = next;
+        }
+    }
+    public class Solution
```
Comment (dsubotkovsky) [inline ScnSoft-Education/LeetCodeTraining/RemoveNthNodeFromEndOfList18/RemoveNthNodeFromEndOfList.cs:14]: add an empty line before declaring the class

File: ScnSoft-Education/LeetCodeTraining.Tests/RemoveNthNodeFromEndOfList.Tests.cs (hunk 1 above)
Comment (CHAriina) [reply ScnSoft-Education/LeetCodeTraining.Tests/RemoveNthNodeFromEndOfList.Tests.cs:10]: adding test

File: ScnSoft-Education/LeetCodeTraining/RemoveNthNodeFromEndOfList18/RemoveNthNodeFromEndOfList.cs (hunk 2 above)
Comment (CHAriina) [reply ScnSoft-Education/LeetCodeTraining/RemoveNthNodeFromEndOfList18/RemoveNthNodeFromEndOfList.cs:14]: adding an empty line

File: ScnSoft-Education/LeetCodeTraining/RemoveNthNodeFromEndOfList18/RemoveNthNodeFromEndOfList.cs [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+            ListNode first = dummy;
+            ListNode second = dummy;
+
+            for (int count = 0; count <= n; count++)
+            {
+                if (first == null) return head; 
+                first = first.next;
+            }
+
+            if (first == null) return head;
```
Comment (dsubotkovsky) [inline ScnSoft-Education/LeetCodeTraining/RemoveNthNodeFromEndOfList18/RemoveNthNodeFromEndOfList.cs]: don't volatile your code style

Comment (CHAriina) [reply ScnSoft-Education/LeetCodeTraining/RemoveNthNodeFromEndOfList18/RemoveNthNodeFromEndOfList.cs]: fixed the code style

**Pull request merged**